include/openthread/ip6.h

```diff
@@ -590,6 +590,24 @@ otError otIp6AddressFromString(const char *aString, otIp6Address *aAddress);
  */
 void otIp6AddressToString(const otIp6Address *aAddress, char *aBuffer, uint16_t aSize);
 
+#define OT_IP6_SOCK_ADDR_STRING_SIZE 48 ///< Recommended size for string representation of an IPv6 socket address.
```

Comment:
Does this include the null-terminator?

Reply:
Is it actually the maxmal size?

Reply:
Yes. `48` should include the null terminator as well. Here is the calculation:
- IP address is max 40 chars (including null).
- We have `[` and `]:`  which adds 3 chars.
- Then the port number as decimal value with max `65536` or 5 chars.
- So total should be `40 + 3 + 5 = 48`.
- Or for `"[1111:2222:3333:4444:5555:6666:7777:8888]:65536"` (which has len 47 and counting null would be 48).